showcase/src/assets/examples/table/with-footer/table.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, ViewEncapsulation } from "@angular/core";
+import { StarkPaginationConfig, StarkTableColumnProperties, StarkTableFilter } from "@nationalbankbelgium/stark-ui";
+
+const DUMMY_DATA: object[] = [
```

Comment:
Could you please reduce this object as we did in another example: https://github.com/NationalBankBelgium/stark/blob/52ae6e0380e24bd3678f0df15edd53746c6ac79c/showcase/src/assets/examples/table/with-custom-styling/table.ts#L4 

```typescript
const DUMMY_DATA: object[] = [
	{ id: 1, cost: 12, description: "number one" },
        /* ... */
       	{ id: 44, cost: 6, description: "the twelfth description" }
];
```

Reply:
Yes, forget it :-), thanks